pkg/smartcontract/manifest/standard/nep24.go

```diff
@@ -0,0 +1,39 @@
+package standard
+
+import (
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract"
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract/manifest"
+)
+
+// Nep24Royalty is a NEP-24 Standard for NFT royalties.
+var Nep24Royalty = &Standard{
+	Manifest: manifest.Manifest{
+		ABI: manifest.ABI{
+			Methods: []manifest.Method{
+				{
+					Name: "royaltyInfo",
+					Parameters: []manifest.Parameter{
+						{Name: "tokenId", Type: smartcontract.ByteArrayType},
+						{Name: "royaltyToken", Type: smartcontract.Hash160Type},
+						{Name: "salePrice", Type: smartcontract.IntegerType},
+					},
+					ReturnType: smartcontract.ArrayType,
+					Safe:       true,
+				},
+			},
+			Events: []manifest.Event{
+				{
+					Name: "RoyaltiesTransferred",
+					Parameters: []manifest.Parameter{
+						{Name: "royaltyToken", Type: smartcontract.Hash160Type},
+						{Name: "royaltyRecipient", Type: smartcontract.Hash160Type},
+						{Name: "buyer", Type: smartcontract.Hash160Type},
+						{Name: "tokenId", Type: smartcontract.ByteArrayType},
+						{Name: "amount", Type: smartcontract.IntegerType},
+					},
+				},
+			},
```

Comment:
NEP-24 itself does not include this event, read the specification. This event belongs to a separate standard like NEP17Payable. Move it to a separate standard.

Reply:
Not resolved.